cpp/src/robotick/workloads/control/SteeringMixerWorkload.cpp

```
@@ -16,6 +16,7 @@ namespace robotick
 		float power_scale_both = 1.0f;
 		float power_scale_left = 1.0f;
 		float power_scale_right = 1.0f;
+		float power_seek_rate = 1.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)